content/SCALE/SCALECLIReference/Auth/CLISessions.md

```diff
@@ -12,4 +12,225 @@ tags:
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### ID Command
+
+The `id` command displays the IDs of all TrueNAS sessions since boot. 
+
+{{< expand "Identifying sessions" "v" >}}
+The `id` command has no required options. It returns a table of recent and active session IDs.
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions id`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions id
++--------------------------------------+
+| id                                   |
++--------------------------------------+
+| 8a21fefe-1255-42eb-a86d-8a7f74752b94 |
+| f01274e8-4954-4926-9346-76a16d3648dd |
++--------------------------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Current Command
+
+The `current` command displays whether or not the TrueNAS sessions in the `sessions` list are current. 
+
+{{< expand "Displaying if sessions are current" "v" >}}
+The `current` command has no required options. It returns a table showing which sessions are current. The rows of the table are relative to the rows returned from the `sessions` command.
+
+*Possible states:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true`   | Indicates the session is current (active). |
+| `false`  | Indicates the session is not current (inactive). |
+{{< /truetable >}}
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions current`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions current
++---------+
+| current |
++---------+
+| false   |
+| true    |
++---------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Internal Command
+
+The `internal` command displays whether sessions in the `sessions` list are internally created (via the web UI) or not (via SSH).
+
+{{< expand "Displaying if sessions are internal" "v" >}}
+The `internal` command has no required options. It returns a table showing if sessions are internal or not. The rows of the table are relative to the rows returned from the `sessions` command.
+
+*Possible states:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true`   | Indicates the session is internal (created via the web UI). |
+| `false`  | Indicates the session is not internal (created via SSH). |
+{{< /truetable >}}
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions internal`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions internal
++----------+
+| internal |
++----------+
+| false    |
+| true     |
++----------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Origin Command
+
+The `origin` command displays the login origin of the sessions in the `sessions` list.
+
+{{< expand "Displaying the login origin of sessions" "v" >}}
+The `origin` command has no required options. It returns a table showing the login origin of each session. The rows of the table are relative to the rows returned from the `sessions` command.
```

Comment:
same comment above. Include the "enter the command, then press Enter line per the template.